### PR TITLE
DEV 2400 gitlab ci

### PR DESCRIPTION
**Pull request by qiaouchicago**
## Ticket Number

dev-2400

## Environment to be used in testing

- [x] `prod`
- [x] `dev-oicr`
- [x] `qa` (which version?)

### CODE REVIEW

File: .gitlab-ci.yml [hunk 1]
```diff
@@ -0,0 +1,95 @@
+include:
+  - project: nci-gdc/gitlab-templates
+    ref: master
```
Comment (stilesj-uchicago) [inline .gitlab-ci.yml:3]: Isn't this usually a version?

File: portal.Dockerfile [hunk 2]
```diff
@@ -1,22 +1,23 @@
 ARG registry=docker.osdc.io
+ARG NPM_REGISTRY="https://registry.npmjs.org/"
```
Comment (stilesj-uchicago) [inline portal.Dockerfile:2]: In the build notes I had captured
```Node v8, npm v5
nvm install 8
nvm use 8
npm i -g npm@5
```
I am surprised to see node:13 and no mention of npm.... but if it works it works.

**Review by stilesj-uchicago — APPROVED** [review]

File: .gitlab-ci.yml (hunk 1 above)
Comment (wrkhenddher) [reply .gitlab-ci.yml:3]: +1

**Review by wrkhenddher — APPROVED** [review]

File: portal.Dockerfile (hunk 2 above)
Comment (qiaouchicago) [reply portal.Dockerfile:2]: I think the docker image we are using have npm 6.14.4

```
❯ docker run --rm -it -v "$PWD":/app -w /app docker.osdc.io/node:13 sh
# bash
root@4c8db8a850ed:/app# npm --version
6.14.4
```

**Pull request merged**